Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/ByAsset.tsx

```diff
@@ -370,7 +370,7 @@ const ByAsset: Application.Types.iByComponent = (props) => {
                         </>
                 }
             </div>
-            <div className='row'>
+            <div style={{ width: '100%'}}>
                 <Paging Current={page + 1} Total={pageInfo.NumberOfPages} SetPage={(p) => setPage(p-1)} />
             </div>
```

Comment:
```suggestion
  <div className="row">
                    <div className="col">
                         <Paging Current={page + 1} Total={pageInfo.NumberOfPages} SetPage={(p) => setPage(p-1)} />
                    </div>
                </div>
```

There is some other stuff associated with `row` including logic related to screen sizing. `ByProcessedFiles` uses `row` and `col` which seems to be the cleaner solution